tests/unit_tests/test_config.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+
+import pytest
+import logging
+from pathlib import Path
+
+import simtools.config as cfg
+from simtools.config import ParameterNotFoundInConfigFile
+
+
+logging.getLogger().setLevel(logging.DEBUG)
+
+
+def test_get_parameters(cfg_setup, configuration_parameters):
+
+    for key, value in configuration_parameters.items():
+        assert value == cfg.get(key, False)
+
+
+def test_get_non_existing_parameter(cfg_setup):
+
+    message = "Configuration file does not contain an entry for the parameter NonExistingEntry"
+    with pytest.raises(ParameterNotFoundInConfigFile,
+                       match=message):
+        cfg.get("NonExistingEntry")
+
+
+def test_input_options(tmp_test_directory):
+    configfile = str(tmp_test_directory) + "/config-test.yml"
+    cfg.setConfigFileName(configfile)
+    assert configfile == cfg.CONFIG_FILE_NAME
+
+
+def test_find_file(cfg_setup, tmp_test_directory):
+
+    tmp_mirror_file = tmp_test_directory / "resources/mirror_MST_D80.dat"
```

Comment:
I wasn't aware that this is possible and is actually nicer than `join`. Now the question is whether we want to be consistent or not. I don't think we should work hard and modify everywhere just to be consistent, but maybe slowly while we code we can change here and there to use this syntax?

Reply:
Done